Core/CoreUITests/CoreUITestCase.swift

```diff
@@ -185,7 +185,7 @@ open class CoreUITestCase: XCTestCase {
         if data.isEmpty {
```

Comment:
```suggestion
        if data.isEmpty || data == "null".data(using: .utf8) {
```

Reply:
Ah, perhaps this will fix the problem I saw. I'll commit these.